<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>JournalUI</class>
 <widget class="QWidget" name="JournalUI">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>842</width>
    <height>664</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Журнал</string>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <item row="0" column="1">
    <widget class="QPushButton" name="m_reviewHistogram_pb">
     <property name="text">
      <string>Гистограмма</string>
     </property>
    </widget>
   </item>
   <item row="0" column="2">
    <widget class="QPushButton" name="m_copy_pb">
     <property name="text">
      <string>Copy</string>
     </property>
    </widget>
   </item>
   <item row="1" column="2">
    <widget class="QPushButton" name="m_past_pb">
     <property name="text">
      <string>Past</string>
     </property>
    </widget>
   </item>
   <item row="2" column="0" colspan="3">
    <widget class="QTableView" name="m_journal_tv">
     <property name="styleSheet">
      <string notr="true"/>
     </property>
    </widget>
   </item>
   <item row="1" column="1">
    <widget class="QPushButton" name="m_clear_pb">
     <property name="text">
      <string>Очистить</string>
     </property>
    </widget>
   </item>
   <item row="0" column="0">
    <widget class="QLabel" name="label">
     <property name="text">
      <string>статистика пользователя</string>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
